src/Creitive/Breadcrumbs/Breadcrumbs.php

```diff
@@ -395,10 +395,11 @@ public function removeAll()
 	 *
 	 * @param  string  $name
 	 * @param  string  $href
+	 * @param  number  $position
 	 * @param  boolean $isLast
 	 * @return string
 	 */
-	protected function renderCrumb($name, $href, $isLast = false)
+	protected function renderCrumb($name, $href, $position, $isLast = false)
```

Comment:
I'm not sure about this, since it breaks BC in the case when someone extends this class. Or is that usually a non-concern?

Can we move the argument to the end and use `null` by default? This way, it will only break code for people who actually override this particular method, and at least not for the people who just call it internally.